kubecost/values.yaml

```
@@ -814,6 +814,10 @@ networkCosts:
       #      - "20.0.0.0/8"
 
   tolerations: []
+  ## useDefaultAffinity: if true, the affinity will be set to a nodeAffinity that will only schedule
+  ## networkCosts pods to run on nodes that have a topology.kubernetes.io/region label.
+  ## If false, the affinity will be set to the affinity provided in the affinity block.
+  useDefaultAffinity: true
   affinity: {}
```

Instead of creating a new config `useDefaultAffinity`, what do you think about just setting the default right here in the `values.yaml`? Then users have the option to override it or modify it as they please.

```suggestion
  ## The networkCosts pod requires labels "topology.kubernetes.io/region" and
  ## "topology.kubernetes.io/zone" for traffic classification. Set `affinity: {}`
  ## to run the networkCosts pod even in the absence of these labels.
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: topology.kubernetes.io/region
            operator: Exists
          - key: topology.kubernetes.io/zone
            operator: Exists
```